jameda_interface_config.js

```diff
@@ -363,4 +365,8 @@ if (isBrowserFirefoxOrSafari() === 'Safari') {
     ];
 }
 
+if (window.jameda.inst.brandLogoPath) {
```

Comment:
If clause should check all accessors or use optional chaining to prevent a possible error from undefined or null values.
`if (window && window.jameda && window.jameda.inst &&  window.jameda.inst.brandLogoPath) {`
or
`if (window?.jameda?.inst?.brandLogoPath) {`